dashboard-notifications/src/main/java/uk/gov/hmcts/reform/dashboard/controllers/DashboardController.java

```diff
@@ -141,14 +141,33 @@ public ResponseEntity<List<Notification>> getNotificationsByCaseIdentifierAndRol
         return new ResponseEntity<>(notificationsResponse, HttpStatus.OK);
     }
 
+    @PutMapping(path = {
+        "notifications/{unique-notification-identifier}"
+    })
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "OK"),
+        @ApiResponse(responseCode = "401", description = "Not Authorized"),
+        @ApiResponse(responseCode = "400", description = "Bad Request")})
+    public ResponseEntity<Void> recordClick(
+        @PathVariable("unique-notification-identifier") UUID id,
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation
+    ) {
+        log.info(
```

Comment:
Can we change this to `Received UUID for recording click: {}`?

Reply:
done